### PR TITLE
UCT/DC/VERBS: Disable tag offload by default

### PR DESCRIPTION
**Pull request by brminich**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4924/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4928/ for details (Mellanox internal link).

File: src/uct/ib/dc/verbs/dc_verbs.h [hunk 1]
```diff
@@ -25,6 +25,13 @@
     }
 
 
+#if IBV_EXP_HW_TM
+#  define UCT_DC_VERBS_TM_ENABLE_STR  "TM_ENABLE=n"
```
Comment (yosefe) [inline src/uct/ib/dc/verbs/dc_verbs.h]: maybe put this #ifdef in dc_verbs.c directly? why need another preprocessor define?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4936/ for details (Mellanox internal link).

**Pull request merged**